src/views/tutorial-view/utils/mdx-components.tsx

```diff
@@ -0,0 +1,25 @@
+import codeBlockPrimitives from '@hashicorp/react-code-block/mdx'
+import Tabs, { Tab } from '@hashicorp/react-tabs' // @TODO use the new tabs within dev-portal, once we figure out groups
+/**
+ * @TODO move over these components from learn, update to new spec
+ *
+ * import Accordion from 'components/accordion'
```

Comment:
Not copying these components over for now. We'll do them one by one once they're ready for dev.